provider/datagen/src/transform/cldr/units/mod.rs

```diff
@@ -118,17 +128,11 @@ impl DataProvider<UnitsConstantsV1Marker> for crate::DatagenProvider {
                 .into_iter()
                 .map(|(cons_name, (num, den, constant_exactness))| {
                     // Converting slices to fraction
```

Comment:
nit: these comments are kind of pointless, the function calls say the same thing

Reply:
done